core/debug/inc/debug.h

```diff
@@ -33,6 +44,10 @@ void debug_fault_mpu(void);
 void debug_fault(THaltError reason, uint32_t lr, uint32_t sp);
 void debug_mpu_config(void);
 void debug_map_addr_to_periph(uint32_t address);
+void debug_register_box(const TUvisorDebugDriver * const driver);
```

Comment:
@AlessandroA I agree with @Patater here - 'debug_register_driver' is more appropriate as the user registers a non-uVisor aware debug drivers. It's just pure coincidence that the driver runs in a debug box and that box requires additional ACLs to do that call.